doc/webservices/extract_api.rst

```diff
@@ -0,0 +1,478 @@
+                        "feature_name_1": validation_1,
+                        "feature_name_2": validation_2,
+                        ...
+                    }
+            },
+            "id": string (hex),
+        }
+
+``validation`` 
+''''''''''''''
+
+validation has the following format for all the features (except for global_taxes and invoice_lines):
+
+.. rst-class:: setup doc-aside
+
+.. switcher::
+
+    .. code-block:: json
+
+        { "content": string|float }
+
+``global_taxes``:
+'''''''''''''''''
+
+**content** is a list of dictionaries. Each dictionary represents a tax:
+
+* ``amount``: amount on which the tax is applied.
+* ``tax_amount``: amount of the tax.
+* ``tax_amount_type``: indicates if the ``tax_amount`` is a percentage or a fixed value. 
+* ``tax_price_include``: indicates if ``amount`` already contains the tax or not. 
+
+.. rst-class:: setup doc-aside
+
+.. switcher::
+
+    .. code-block:: json
+
+        {"content": [
+            {
+                "amount": float,
+                "tax_amount": float,
+                "tax_amount_type": "fixed"|"percent",
+                "tax_price_include": bool
+            },
+            ...
+        ]}
+
+``invoice_lines``:
+''''''''''''''''''
+
+**lines** is a list of dictionaries. Each dictionary represents an invoice line. The dictionary keys speak for themselves.
+
+.. rst-class:: setup doc-aside
+
+.. switcher::
+
+    .. code-block:: json
+
+        {"lines": [
+            {
+                "description": string,
+                "quantity": float,
+                "unit_price": float,
+                "product": string,
+                "taxes_amount": float,
+                "taxes": [
+                            {
+                                "amount": float, 
+                                "type": "fixed"|"percent", 
+                                "price_include": bool
+                            },
+                            ...
+                        ],
+                "subtotal": float,
+                "total": float
+            },
+            ...
+        ]}
+
+Response
+^^^^^^^^
+
+* ``jsonrpc``:  Same as for :ref:`/parse <webservices/extract_api/invoice_parse>`.
+* ``id``:  Same as for :ref:`/parse <webservices/extract_api/invoice_parse>`.
+* ``result``: 
+    * ``status_code``: code indicating the status of the request. ``status_code`` is 0 in case of success. Other ``status_code`` are detailed in the table below.
+    * ``status_msg``: A string giving verbose details about the request status.
+
+=============  ==========================================================
+ status_code       status_msg
+=============  ==========================================================
+0              Success
+12             Validation format is incorrect
+=============  ==========================================================
+
+The ``status_code`` 12 is only used in :ref:`integration mode <webservices/extract_api/integration_testing>`
```

Comment:
No you don't need to validate all the features. I will precise that. I should remove this line since status code since it is also used in "normal" mode now.